packages/directlinespeech/__tests__/utilities/createTestHarness.js

```diff
@@ -5,11 +5,12 @@ import createQueuedArrayBufferAudioSource from './createQueuedArrayBufferAudioSo
 import fetchSpeechCredentialsWithCache from './fetchSpeechCredentialsWithCache';
 import fetchSpeechData from './fetchSpeechData';
 
-export default async function createTestHarness() {
+export default async function createTestHarness(enableInternalHttpSupport = false) {
```

Comment:
I think we should make this an object, so we can easily add more option parameters down the road.

```suggestion
export default async function createTestHarness({ enableInternalHttpSupport = false }) {
```

Reply:
Agree with @tdurnford. Also with casing, acronyms are always in uppercase (except id, which is not an acryonym): `elementId`, `getElementById`, `encodeURI`, `createCDATASection`.

```suggestion
export default async function createTestHarness({ enableInternalHTTPSupport = false }) {
```